src/main/java/com/yurjinia/common/handlers/LogoutHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.yurjinia.common.handlers;
+
+import com.yurjinia.common.exception.CommonException;
+import com.yurjinia.common.exception.ErrorCode;
+import com.yurjinia.common.security.jwt.constants.JwtConstants;
+import com.yurjinia.common.security.jwt.service.JwtBlacklistService;
+import com.yurjinia.common.security.jwt.service.JwtService;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.logout.SecurityContextLogoutHandler;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+@Component
+@RequiredArgsConstructor
+public class LogoutHandler {
+
+    private final JwtService jwtService;
+    private final JwtBlacklistService jwtBlacklistService;
```

Comment:
I'm thinking that for making less dependency injections - do one dependency injection of `JwtBlacklistService` within `JwtService`, add respective methods in `JwtService` and then when you need to use exactly the logic related to blacklisting - call it from `JwtService`.